tests/integration/golang/fixtures/run.go

```diff
@@ -119,3 +154,49 @@ func (f RunFixtures) FindMinMaxRowNums(
 	}
 	return int64(min), int64(max), nil
 }
+
+// CreateTag creates a new Tag for a run
+func (f RunFixtures) CreateTag(
+	ctx context.Context, tag models.Tag,
+) error {
+	if err := f.tagRepository.CreateRunTagWithTransaction(ctx, f.db, tag.RunID, tag.Key, tag.Value); err != nil {
+		return eris.Wrap(err, "error creating run tag")
+	}
+	return nil
+}
+
+func (f RunFixtures) CreateMetrics(
```

Comment:
can you please add comments? every comment must end with a dot.

Reply:
done